common.yml

```diff
@@ -1860,10 +1836,10 @@ buildvariants:
     run_kinit: true
   tasks: *rhel71_ppc64le_tasks
 
-- name: rhel82-ppc64le
-  display_name: ZAP PPC64LE RHEL 8.2
+- name: rhel81-ppc64le
```

Comment:
@huan-Mongo was there a particular reason you had for not using the rhel81 distro here?

Reply:
In 8.1 image build, I got such error ```fork/exec /opt/mongodbtoolchain/v3/bin/ppc64le-mongodb-linux-gcc: no such file or directory
```
Seems toolchain is missing
https://evergreen.mongodb.com/task/mongo_tools_rhel81_ppc64le_dist_patch_37b7176480b317a7b97077ee2297397df922cf0b_5e5839bc850e6122204d8ce9_20_02_27_21_50_53

Reply:
ah, I see. Good catch!